src/lang/wasm.ts

```diff
@@ -92,12 +92,20 @@ export type { Solid } from '../wasm-lib/kcl/bindings/Solid'
 export type { KclValue } from '../wasm-lib/kcl/bindings/KclValue'
 export type { ExtrudeSurface } from '../wasm-lib/kcl/bindings/ExtrudeSurface'
 
+// The first two items are the start and end points (byte offsets from the start of the file).
+// The third item is whether the source range belongs to the 'main' file, i.e., the file currently
+// being rendered/displayed in the editor (TODO we need to handle modules better in the frontend).
 export type SourceRange = [number, number, boolean]
```

Comment:
If you use this style of multi-line comment, it's treated as a doc comment, at least by some editors.

```typescript
/**
 * Return the representation that can be serialized to JSON.  This should only
 * be used within this module.
 */